python/private/toolchains_repo.bzl

```
@@ -384,6 +386,16 @@ toolchain_aliases repo because referencing the `python` interpreter target from
 this repo causes an eager fetch of the toolchain for the host platform.
     """,
     attrs = {
+        "archs": attr.string_dict(
```

nit: 
```suggestion
        "arch_names": attr.string_dict(
```

or `cpu_names`.